cmp/report.go

```diff
@@ -24,8 +24,8 @@ type defaultReporter struct {
 func (r *defaultReporter) PushStep(ps PathStep) {
 	r.curPath.push(ps)
 }
-func (r *defaultReporter) Report(f reportFlags) {
-	if f&reportUnequal > 0 {
+func (r *defaultReporter) Report(f ReportFlags) {
+	if f&ReportUnequal > 0 {
```

Comment:
Optional. I didn't call this out on previous CL, but I tend to view single bit mask comparison as != 0 or == 0. > 0 seems to imply more values.